src/clj/game/cards/assets.clj

```diff
@@ -232,8 +232,12 @@
              {:prompt "Pay 1 [Credits] or take 1 tag" :choices ["Pay 1 [Credits]" "Take 1 tag"]
               :player :runner :msg "make the Runner pay 1 [Credits] or take 1 tag"
               :delayed-completion true
-              :effect (req (if-not (and (= target "Pay 1 [Credits]") (pay state side card :credit 1))
-                             (do (tag-runner state side eid 1) (system-msg state side "takes 1 tag"))
+              :effect (req (if-not (and (= target "Pay 1 [Credits]")
+                                        (pay state side card :credit 1)
+                                        (effect-completed state side eid))
+                             (do (tag-runner state side eid 1)
+                                 (system-msg state side "takes 1 tag")
+                                 (effect-completed state side eid))
```

Comment:
This `effect-completed` is unnecessary, since the `eid` passed to `tag-runner` will "complete" the City Surv. effect once `tag-runner` is complete (which accounts for tag prevention).